osu.Game/Screens/OnlinePlay/Matchmaking/Match/ScreenMatchmaking.HistoryFooterButton.cs

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Allocation;
+using osu.Framework.Graphics.Sprites;
+using osu.Game.Graphics;
+using osu.Game.Online.Multiplayer;
+using osu.Game.Screens.Footer;
+
+namespace osu.Game.Screens.OnlinePlay.Matchmaking.Match
+{
+    public partial class ScreenMatchmaking
+    {
+        private partial class HistoryFooterButton : ScreenFooterButton
+        {
+            [Resolved]
+            private OsuGame? game { get; set; }
+
+            private readonly MultiplayerRoom room;
+
+            public HistoryFooterButton(MultiplayerRoom room)
+            {
+                this.room = room;
+
+                Action = openRoomHistory;
+            }
+
+            [BackgroundDependencyLoader]
+            private void load(OsuColour colours)
+            {
+                Text = "History";
+                Icon = FontAwesome.Solid.Globe;
+                AccentColour = colours.Lime1;
+            }
+
+            private void openRoomHistory()
+                => game?.OpenUrlExternally($@"/multiplayer/rooms/{room.RoomID}");
```

Comment:
This will open this page: https://dev.ppy.sh/multiplayer/rooms/6440.

To me, "room history" means https://dev.ppy.sh/multiplayer/rooms/6440/events. Did you intentionally choose the first one over the other?

As an aside, the full room history is weirdly broken for reasons I don't get. There are two "maps started" that I never started (the map there is just the lowest star rating in the pool), and it seems like quick play may be creating double the necessary number of playlist items...?